content/applications/sales/crm/track_leads/lead_scoring.rst

```diff
@@ -2,77 +2,81 @@
 Assign leads based on scoring
 =============================
 
-With *Leads Scoring* you can automatically rank your leads based on
-selected criteria.
+.. important::
+   This feature will be completely replaced by **Predictive Lead Scoring** with Version 14.3's
+   release.
 
-For example you could score customers from your country higher or the
-ones that visited specific pages on your website.
+With *Lead Scoring* you can automatically rank your leads based on selected criteria.
+
+For example, you could score customers higher if coming from the same country as your company over a
+customer from a different country.
 
 Configuration
 =============
 
-To use scoring, install the free module *Lead Scoring* under your
-*Apps* page (only available in Odoo Enterprise).
+To use scoring, go to :menuselection: 'Apps', remove the *Apps* filter, search for *Lead Scoring*,
```

Comment:
Did this instead since **Apps** there is technically a menu item I believe:

```suggestion
To use scoring, go to :menuselection:`Apps`, remove the :guilabel:`Apps` filter, search for
:guilabel:`Lead Scoring` and then install the module.
```